Tests/Functional/Frontend/AbstractTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Tea\Tests\Functional\Frontend;
+
+use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
+
+/**
+ * Provides general set up for frontend related functional tests.
+ */
+abstract class AbstractTest extends FunctionalTestCase
```

Comment:
As far as I understand it, this is the base class for the controller tests. So I suggest the name `AbstractControllerTest`, and move it to `Tests/Functional/Controller`.

Reply:
Well it is related for all frontend related stuff. We have some extensions where we provide ViewHelper, UserFunc, Middleware or other stuff not necessarily related to controller. Controller are a bit to "deep knowledge" regarding testing a request for my taste.
That's why I used 'Frontend' as folder and namespace.

But I'm fine to adjust this, especially as I don't see that EXT:tea will get anything like that soon.